hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -157,6 +157,13 @@ objects:
         image: ${REPO_DIGEST}
         displayName: splunk-forwarder-operator Registry
         publisher: SRE
+        grpcPodConfig:
```

Comment:
There's already a `grpcPodConfig` on line 155. Can you please move this up there?